pkg/fftypes/enum.go

```diff
@@ -54,3 +54,16 @@ func (ts *FFEnum) UnmarshalText(b []byte) error {
 	*ts = FFEnum(strings.ToLower(string(b)))
 	return nil
 }
+
+func FFEnumParseString(t string, i string) FFEnum {
+	e, ok := enumValues[t]
```

Comment:
```suggestion
	e, ok := enumValues[strings.ToLower(t)]
```

Our standard semantic is case insensitivity. I think we

Reply:
Also, should we have a standard error for the case that it is non-empty, but not one of the valid values?